eng/common/TestResources/New-TestResources.ps1

```diff
@@ -735,76 +746,73 @@ try {
         'PowerShell', '${{env:{0}}} = ''{1}'''
     }
 
-    # Deploy the templates
-    foreach ($templateFile in $templateFiles) {
```

Comment:
All the changes from here down are just me removing the `foreach` loop and removing one level of indentation. The diff doesn't make this clear.

Reply:
Tip: it does if you hide whitespace in the PR options. 😉

Reply:
Mind blown